### PR TITLE
I11 breadcrumbs

### PR DESCRIPTION
**Pull request by vrcode-dev**
merging to develop branch

### CODE REVIEW

File: src/components/Breadcrumbs/Breadcrumbs.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+const Breadcrumbs = (props) => {
+  const {
+    history,
+    location: { pathname },
+  } = props;
+  const pathnames = pathname.split("/").filter((x) => x);
+  return (
+    <div className="avenir pa2 ph5 link flex">
+      {pathnames.length > 0 && (
```
Comment (tz-goat) [inline src/components/Breadcrumbs/Breadcrumbs.js]: I'd suggest moving `pathnames.length > 0` to the top level so it won't take extra space when there's no path

Comment (Iancam) [reply src/components/Breadcrumbs/Breadcrumbs.js]: Yup, the code could be a bit easier to read if you assigned the different high level parts to variables. I think this part is the home crumb?

File: src/components/Breadcrumbs/Breadcrumbs.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <div>
+          <Link
+            className="no-underline avenirlink hover-green hover-animate"
```
Comment (Iancam) [inline src/components/Breadcrumbs/Breadcrumbs.js]: avenirlink -> avenir link (do a project wide search)

File: package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@reduxjs/toolkit": "^1.5.0",
+    "@material-ui/core": "^4.11.2",
```
Comment (Iancam) [inline package.json]: but why? If we aren't usin' it, we should toss it.

**Comment by Iancam** [comment]
First of all, this thing basically works from the router, which makes it easy to
apply to any page. Very modular. 👍 

Actually, it's so modular that I have a recommendation: in App, you could add page title to each element in the routes list, which would be good to have anyway. Then breadcrumbs could be sit like the header–maybe in the route.map
part, so you have title access.

File: package.json (hunk 3 above)
Comment (vrcode-dev) [reply package.json]: got it!

File: src/components/Breadcrumbs/Breadcrumbs.js (hunk 2 above)
Comment (vrcode-dev) [reply src/components/Breadcrumbs/Breadcrumbs.js]: oops, the space must have been lost somewhere along the way!

File: src/components/Breadcrumbs/Breadcrumbs.js (hunk 1 above)
Comment (vrcode-dev) [reply src/components/Breadcrumbs/Breadcrumbs.js]: > I'd suggest moving `pathnames.length > 0` to the top level so it won't take extra space when there's no path

There was no extra space if there was no path. To verify, i tried changing the background color of the breadcrumbs and nothing was rendered if there's no path. 

Nevertheless, i reorganized/simplified the logic and put the conditional statement at top level per suggestions.

**Pull request merged**